event-schema/event_schema.go

```diff
@@ -939,11 +950,17 @@ func (manager *EventSchemaManagerT) populateSchemaVersion(o *OffloadedSchemaVers
 
 // This should be called during the Initialize() to populate existing event Schemas
 func (manager *EventSchemaManagerT) populateEventSchemas() {
+	defer setEventSchemasPopulated(true)
```

Comment:
can we add a metric around how much time this step is taking

Reply:
Done.